src/module.c

```diff
@@ -5358,6 +5358,10 @@ int RM_HashSet(RedisModuleKey *key, int flags, ...) {
  * expecting a RedisModuleString pointer to pointer, the function just
  * reports if the field exists or not and expects an integer pointer
  * as the second element of each pair.
+ * 
+ * REDISMODULE_HASH_EXPIRE_TIME: retrieves the expiration time of a field in the hash.
+ * The function expects a `mstime_t` pointer as the second element of each pair.
+ * If the field exists but has no expiration time, `REDISMODULE_NO_EXPIRE` is set. 
```

Comment:
Maybe we should document that this can not be used with combination of other flags. Or if there are flag that can be used with it we should document it also.